scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py

```diff
@@ -193,34 +193,36 @@ def create_position_table(fq_temp_table_dataset, min_variant_samples):
 def make_new_pet_union_all(fq_pet_vet_dataset, fq_temp_table_dataset, sample_ids):
   def get_pet_subselect(fq_pet_table, samples, id):
     sample_stanza = ','.join([str(s) for s in samples])
-    sql = f"    q_{id} AS (SELECT p.location, p.sample_id, p.state from `{fq_pet_table}` p " \
-          f"    join `{fq_temp_table_dataset}.{VET_DISTINCT_POS_TABLE}` v on (p.location = v.location) WHERE p.sample_id IN ({sample_stanza})), "
+    sql = f"    q_{id} AS (SELECT p.location, p.sample_id, p.state FROM `{fq_pet_table}` p \n" \
+          f"        JOIN `{fq_temp_table_dataset}.{VET_DISTINCT_POS_TABLE}` v on (p.location = v.location) \n WHERE p.sample_id IN ({sample_stanza})), "
     return sql
 
-  subs = {}
   for i in range(1, PET_VET_TABLE_COUNT+1):
-    partition_samples = get_samples_for_partition(sample_ids, i)
+    partition_samples = get_samples_for_partition(sample_ids, i)  #sample ids for the partition
 
-      # KCIBUL -- grr, should be fixed width
-    fq_pet_table = f"{fq_pet_vet_dataset}.{PET_TABLE_PREFIX}{i:03}"
     if len(partition_samples) > 0:
+      subs = {}
+      create_or_insert = f"\nCREATE OR REPLACE TABLE `{fq_temp_table_dataset}.{PET_NEW_TABLE}` {TEMP_TABLE_TTL} AS \n WITH \n" if i == 1 \
+      else f"\nINSERT INTO `{fq_temp_table_dataset}.{PET_NEW_TABLE}` \n WITH \n"
+      fq_pet_table = f"{fq_pet_vet_dataset}.{PET_TABLE_PREFIX}{i:03}"
```

Comment:
if the \n's are all for readability, would it make sense to space them that way in this code? like end each line here with \n ?

Reply:
does there need to be a name after AS ?

Reply:
Both the line returns and the "AS" with no alias were copied from the original query and are in other queries.  When I added an alias it failed, so I'm going to keep this one that way.

Reply:
huh! well I can't argue with that logic....